public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -135,15 +135,22 @@ interface NotifiersTableProps {
 }
 
 function LastNotify({ lastNotify }: { lastNotify: string }) {
-  const lastNotifyDuration = lastNotify;
-  return (
-    <Stack alignItems="center">
-      <div>{dateTime(lastNotifyDuration).locale('en').fromNow(true)} ago</div>
-      <Icon name="clock-nine" />
-      <div>{dateTimeFormat(lastNotifyDuration, { format: 'YYYY-MM-DD HH:mm:ss' })}</div>
-    </Stack>
-  );
+  const lastNotifyDate = lastNotify;
```

Comment:
What's the reason for adding this variable? can't you use `lastNotify` directly? Or rename the parameter.

Reply:
You're completelly right!!!